app/src/main/java/com/github/se/project/ui/lesson/EditRequestedLesson.kt

```diff
@@ -40,37 +42,38 @@ fun EditRequestedLessonScreen(
 
   val context = LocalContext.current
 
-  /**
-   * Lambda function to handle lesson updates. Updates the lesson details and navigates back to the
-   * home screen upon completion.
-   *
-   * @param lesson The updated lesson to be saved.
-   */
   val onConfirm = { lesson: Lesson ->
-    // Updates the lesson in the ViewModel and refreshes the user's lesson list
+    /**
+     * Lambda function to handle lesson updates. Updates the lesson details and navigates back to
+     * the home screen upon completion.
+     *
+     * @param lesson The updated lesson to be saved.
+     */
```

Comment:
I am not sure why this comment block has been moved. I think we should put comments above the function it is related to, not inside.